File: app/src/main/java/com/example/farith/dailynotes/ModelClass/NoteClass.java
package com.example.farith.dailynotes.ModelClass;

import android.provider.BaseColumns;

public class NoteClass implements BaseColumns {
    public final static String TABLE_NAME = "notes";

    public final static String ID = BaseColumns._ID;
    public final static String NOTE_CONTENT = "note_content";
    public static final String DATE = "date";
    public static final String NOTIFICATION_ID = "notification_id";
    public static final String REMINDER_TIME = "reminder_time";
}

File: app/src/main/java/com/example/farith/dailynotes/modelClass/NoteClass.kt
package com.example.farith.dailynotes.modelClass

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import java.io.Serializable

@Entity(tableName = "notes")
data class NoteClass(
    @ColumnInfo(name = "note_content")
    var noteContent: String?,
    @ColumnInfo(name = "date")
    var date: String?,
    @PrimaryKey(autoGenerate = false)
    var notificationId: Int?,
    @ColumnInfo(name = "reminder_time")
    var reminderTime: String?
) : Serializable {

}